similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultMatchedPatientClusterView.java

```diff
@@ -22,6 +22,7 @@
 import org.phenotips.data.similarity.PatientSimilarityView;
 
 import java.util.Collections;
+import java.util.Comparator;
```

Comment:
Another minor fix: should be `since 1.4` not `since 1.2`. Github does not let me comment on that line, it is a few lines below this one

Reply:
Why 1.4?